### ISSUE TITLE
[automation/dotnet] Program output contains ASP.NET debug information

### ISSUE THREAD

**Issue by mikhailshilkov** [pulumi/pulumi#6780]
Working with [Automation API Inline Program in C#](https://github.com/pulumi/automation-api-examples/tree/main/dotnet/InlineProgram), I get a bunch of ASP.NET debug messages in my program's output.

```
$ dotnet run
successfully initialized stack
installing plugins...
plugins installed
setting up config...
config set
refreshing stack...
Refreshing (dev)

View Live: https://app.pulumi.com/mikhailshilkov/auto-cs-inline-azure/dev/updates/1


 
Resources:

Duration: 1s

refresh complete
updating stack...
info: Microsoft.Hosting.Lifetime[0]
      Now listening on: http://0.0.0.0:58218
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
      Content root path: /Users/mikhailshilkov/Work/play/auto-api-azure-cs
Updating (dev)

View Live: https://app.pulumi.com/mikhailshilkov/auto-cs-inline-azure/dev/updates/2

info: Microsoft.AspNetCore.Hosting.Diagnostics[1]
      Request starting HTTP/2 POST http://127.0.0.1:58218/pulumirpc.LanguageRuntime/GetRequiredPlugins application/grpc -
info: Microsoft.AspNetCore.Routing.EndpointMiddleware[0]
      Executing endpoint 'gRPC - /pulumirpc.LanguageRuntime/GetRequiredPlugins'
info: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]
      Executed endpoint 'gRPC - /pulumirpc.LanguageRuntime/GetRequiredPlugins'

info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
      Request finished HTTP/2 POST http://127.0.0.1:58218/pulumirpc.LanguageRuntime/GetRequiredPlugins application/grpc - - 200 - application/grpc 96.7268ms
info: Microsoft.AspNetCore.Hosting.Diagnostics[1]
      Request starting HTTP/2 POST http://127.0.0.1:58218/pulumirpc.LanguageRuntime/Run application/grpc -
info: Microsoft.AspNetCore.Routing.EndpointMiddleware[0]
      Executing endpoint 'gRPC - /pulumirpc.LanguageRuntime/Run'
 +  pulumi:pulumi:Stack auto-cs-inline-azure-dev creating 
info: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]
      Executed endpoint 'gRPC - /pulumirpc.LanguageRuntime/Run'
info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
      Request finished HTTP/2 POST http://127.0.0.1:58218/pulumirpc.LanguageRuntime/Run application/grpc - - 200 - application/grpc 1005.6498ms
 +  pulumi:pulumi:Stack auto-cs-inline-azure-dev created 
 
Outputs:
    website_url: "test"

Resources:
    + 1 created

Duration: 2s

info: Microsoft.Hosting.Lifetime[0]
      Application is shutting down...
update summary:
    Create: 1
website url: test
```

Ideally, I wouldn't see it at all.

**Comment by mikhailshilkov** [comment]
A workaround is adding an `appsettings.json` file to the project:

```json
{
  "Logging": {
    "LogLevel": {
      "Default": "Warning",
      "Microsoft": "Warning",
      "Microsoft.Hosting.Lifetime": "None",
      "Microsoft.AspNetCore.Hosting.Diagnostics": "None"
    },
    "EventLog": {
      "LogLevel": {
        "Microsoft.AspNetCore.Hosting.Diagnostics": "None"
      }
    }
  }
}
```

It would be nice not to require doing so.

**Comment by orionstudt** [comment]
It's logging from the inline program host. I agree, it shouldn't be there. Idk if we want to provide a way to enable it conditionally or hook into it though

**Comment by orionstudt** [comment]
FYI I believe this can be closed because of #7023
